lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -113,18 +118,53 @@ def human_readable(size, isbits=False, unit=None):
     ''' Return a human readable string '''
     try:
         return basic.bytes_to_human(size, isbits, unit)
-    except:
+    except Exception as e:
         raise errors.AnsibleFilterError("human_readable() can't interpret following string: %s" % size)
 
 
 def human_to_bytes(size, default_unit=None, isbits=False):
     ''' Return bytes count from a human readable string '''
     try:
         return basic.human_to_bytes(size, default_unit, isbits)
-    except:
+    except Exception as e:
```

Comment:
e is unsued, just `except Exception:` to avoid 'bare exceptions'

Reply:
Fixed.